### ISSUE TITLE
Add option to map tablets across monitors using input:tablet:output=ALL in hyprland.conf

### ISSUE THREAD

**Issue by Da-Boom** [hyprwm/Hyprland#6889]
### Description

In relation to The changes made in #5902 and the bug at #6023

Its well known that Hyprland gives the option `input:tablet:output=`  which allows me to specify a monitor to bind a tablet to. This is great for regular drawing tablets, as it simplifies things. By default, when not specified it maps to whichever monitor the cursor is currently on. 

But there's a situation where this setup not great. (not counting the obvious oddly shaped tablets and monitors like a 16:10 tablet on a 16:9 display, which has obvious implications, but can also be manipulated with other input settings for tablets.)

when i have an abnormal pointing devices that doesn't follow these conventions. like in the case of `wlx-overlay-s` made by galister (https://github.com/galister/wlx-overlay-s) or say if someone were to try using a wii remote as a pointing device across monitors - (`cwiid` is a thing)

`wlx-overlay-s` designed to work with `wlroots` based compositors. It allows using desktop portals to preview the screens onto surfaces on a HMD (head mounted display) it simulates a pointer device using `uinput` and maps using the regular tablet surface - which is defined by `input:tablet:output=`  in hyprland. 

This leads to a problem. Given that it expects to get the entire desktop canvass for BOTH monitors, as that's what most other compositors do. it ends up inadvertently mapping one monitor across both the virtual screens in vr. 

 - This leads to me, with my cursor on the right screen, pointing my VIVE wand at the left, and finding the cursor moving about on the left half of the right screen.  If i point at the right screen, i get control of the right half of the right screen.
 
 i run 2 1440p displays (resulting resolution 5120x1440). i imagine it would behave even more wacky if they were different resolutions, or if i had a third display.
 
what i want in this feature request is the simple ability to supply `ALL` to the `input:tablet:output=` setting, allowing me to map the virtual device across all existing separate displays at the same time.
 
I am willing to test this as i have my `monado` setup with galisters' patches functioning nicely, a HMD and i already have `wlx-overlay-s` installed.

**Comment by CobaltSpace** [comment]
I was doing some testing. Using my drawing tablet, I tested the following:
```hyprlang
input:tablet {
  output = $top_left_output
  region_size = $layout_width, $layout_height
}
```
This worked for my tablet, but when I launched wlx-overlay-s, it was still broken. I then tried
```hyprlang
device {
  name = wlxoverlay-s-keyboard-mouse-hybrid-thing-1
  output = $top_left_output
  region_size = $layout_width, $layout_height
}
```
but this did not work either. So part of the issue is that while wlx-overlay-s acts like a tablet, if you run `hyprctl devices`, it shows up under mouse and under keyboard.

In conclusion, `input:tablet:output=ALL` wouldn't even fix wlx-overlay-s. 
Something lower needs to be fixed to get the behavior we want.

This was tested on hyprland v0.41.2

I wanted to try v0.42 or -git, but Xwayland would crash when plugging in the headset, so stuck on v0.41.2 for now.
